index.html

```diff
@@ -903,10 +903,29 @@ <h4>Issue Credential</h4>
         </p>
         <p>
 If a provided `credential` already contains one or more proofs,
-an issuing instance SHOULD append its proofs to the list of existing proofs
-provided by the caller, converting a single proof value to an array of
-proof values if necessary. An issuing instance MAY be configured to return an
-error if `credential` values that contain existing `proof` values are provided.
+the behavior is determined by the issuer instance configuration. An issuing instance
+SHOULD be configured to handle existing proofs in one of the following ways:
+        </p>
+        <ul>
+          <li>
+            <strong>Proof Sets</strong>: Append their proofs to the list of existing proofs 
+provided by the caller, converting a single proof value to an array of proof values 
+if necessary, creating a proof set where multiple proofs exist in parallel.
+          </li>
+          <li>
+            <strong>Proof Chains</strong>: Append their proofs to create a proof chain
+where proofs are linked in a specific sequence, potentially using properties like
+<code>previousProof</code> to establish the chain relationship.
```

Comment:
The word "potentially" can be removed here because a proof chain requires the `previousProof` property; it is what establishes that a proof change is operative. Also some adjustments per my other change suggestion above:

```suggestion
            <strong>Proof Chains</strong>: Append new proofs to create or extend
an existing proof chain. Here proofs are linked in a specific sequence, using the
<code>previousProof</code> property to establish the chain relationship.
```